packages/app/src/app/pages/Dashboard/Components/Repository/RepositoryListItem.tsx

```diff
@@ -81,12 +80,7 @@ export const RepositoryListItem: React.FC<RepositoryProps> = ({
             </Stack>
           </Column>
           <Column span={[0, 4, 4]} as={Stack} align="center">
-            <Stack
-              justify="flex-end"
-              align="center"
-              gap={2}
-              css={{ paddingRight: '16px' }}
```

Comment:
Removing the `padding-right` leaves no space between the branch count and the dropdown trigger, this becomes quite visible when hovering it:

![Screen Shot 2023-02-01 at 22 38 23](https://user-images.githubusercontent.com/24959348/216209870-45fe579f-3a4e-4cbb-ba20-181dcc52d7f7.png)